tree/ntuple/v7/doc/architecture.md

```diff
@@ -305,7 +305,7 @@ For reading, the `RMiniFile` always uses an `RRawFile`.
 The RRawFile internal abstract class provides an interface to read byte ranges from a file, including vector reads.
 Concrete implementations exist for local files, XRootD and HTTP (the latter two through the ROOT plugin mechanism).
 The local file implementation on Linux uses uring for vector reads, if available.
-A raw file that uses a TFile (to be implemented) provides access to the full set of TFile implementations, e.g. TMemFile.
+A raw file that uses a TFile provides access to the full set of TFile implementations, e.g. TMemFile.
```

Comment:
Does this sentence refer to the `RRawFileTFile` (that is now mentioned under the "Ownership model" section)?

Reply:
Yes; I added an explicit mention.

Reply:
Looks good now, thanks!